src/Components/Navbar.jsx

```diff
@@ -1,12 +1,29 @@
-
+import { Link } from "react-router-dom"
+import { useState, useEffect } from "react";
+import axios from 'axios';
 const Navbar = () => {
+  const [topics, setTopics] = useState([]);
+  useEffect(() => {
+    axios
+      .get(`https://niamh-news.herokuapp.com/api/topics`)
+      .then(({ data }) => {
+        setTopics(data.topics);
+      });
+  }, []);
```

Comment:
This request would be better in a utils folder, so you only have to make it once, then can resuse it if needed